src/main/java/org/mskcc/cbio/oncokb/service/UserService.java

```diff
@@ -667,6 +668,23 @@ public boolean userHasAuthority(User user, String authority) {
         return user.getAuthorities().stream().filter(userAuth -> userAuth.getName().equalsIgnoreCase(authority)).count() > 0;
     }
 
+    public List<UserDTO> getPotentialDuplicateAccountsByUser(UserDTO user) {
+        return searchAccountsForPotentialDuplicateUser(user, getAllManagedUsers());
+    }
+
+    public List<UserDTO> searchAccountsForPotentialDuplicateUser(UserDTO user, List<UserDTO> allUsers) {
+        String userFullName = user.getFirstName() + " " + user.getLastName();
+        JaroWinklerSimilarity jw = new JaroWinklerSimilarity();
+        List<UserDTO> potentialDuplicateUsers = new ArrayList<>();
+        for (UserDTO potentialDuplicate : allUsers) {
+            String potentialDuplicateFullName = potentialDuplicate.getFirstName() + " " + potentialDuplicate.getLastName();
+            if (user.getId() != potentialDuplicate.getId() && jw.apply(userFullName, potentialDuplicateFullName) > .7) {
```

Comment:
Does the params order matter in  `jw.apply`?

Reply:
Nope